new/components/vcloud/configure.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tempfile
+import json
+
+import fabric
+
+import vcloud_plugin_common
+from cloudify import ctx
+
+PROVIDER_CONTEXT_RUNTIME_PROPERTY = 'provider_context'
+
+
+def configure(vcloud_config):
+    """
```

Comment:
just to follow up on the standard, docs string should be written as:

```python
def func():
    """one liner summary

    additional info
    """
```

not a must. just sayin`..